domains/z3us.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SgtZeus",
+        "email": "dbzkronos@gmail.com"
+    },
+    "record": {
+        "TXT": "z3us"
```

Comment:
TXT record alone doesn't do anything.

Reply:
oh sorry i didnt really know what i was doing well ill make the changes